packages/ui/src/components/account/AccountSearchResults.tsx

```diff
@@ -137,7 +137,10 @@ const AccountSearchResults = ({
                             <AccountDisplay
                                 key={account.address}
                                 account={account}
-                                selected={filter === account.address}
+                                selected={
+                                    filter.toLowerCase() ===
```

Comment:
same checksum comment

Reply:
Here I have to check if it's an address first so that toChecksumAddress doesn't throw an error.
`utils.isAddress(filter) &&
                                    toChecksumAddress(filter) ===
                                        account.address`